src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/loadbalancer/LoadBalancerFormula.kt

```diff
@@ -19,6 +20,10 @@ interface LoadBalancerFormula {
         key: SshKey,
         aws: Aws
     ): ProvisionedLoadBalancer
+
+    fun updateJiraConfiguration(
```

Comment:
This an interface in the API package which means this is an SPI. Adding new method without default implementation means API-breaking change. I suggest adding a default no-op implementation.

Reply:
Done.